AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/MutationStateStructures.dfy

```diff
@@ -475,6 +429,8 @@ module {:options "/functionSyntax:4" } MutationStateStructures {
 
     var originalHV := MutablePropertiesJsonToHierarchyVersion(OriginalJson);
     var terminalHV := MutablePropertiesJsonToHierarchyVersion(TerminalJson);
+    // TODO: Generally, the logic of this module is around serialization and deserialization.
```

Comment:
Is this TODO not supposed to follow TODO comment style we do for hv-2/mutation? Does mutation has any TODO comment style like we follow in hv-2 (`TODO-HV-2-M2`)

Reply:
You or I could pick this TODO up right now;
I think I left this here to talk about it with you or José.